### ISSUE TITLE
Customizable Haptic Feedback and Amoled Theme

### ISSUE THREAD

**Issue by TISupe** [Swordfish90/Lemuroid#919]
(✿╹◡╹) Please add Customizable Haptic Strength and Amoled theme too please please please(•ˋ _ ˊ•)

**Comment by RainbowShatter** [comment]
Definitely. And if they aren't going to do it I am going to contribute for that, I am going to start my development career and make everyone happy with all of the requested features, however, I am too occupied with all of my music, when I finish my first album I will finally give the feature you want

**Comment by TISupe** [comment]
> Definitely. And if they aren't going to do it I am going to contribute for that, I am going to start my development career and make everyone happy with all of the requested features, however, I am too occupied with all of my music, when I finish my first album I will finally give the feature you want

W (^-^) (´｡･v･｡｀)(´｡･v･｡｀) im learning too.  ill try too

**Comment by RainbowShatter** [comment]
> > Definitely. And if they aren't going to do it I am going to contribute for that, I am going to start my development career and make everyone happy with all of the requested features, however, I am too occupied with all of my music, when I finish my first album I will finally give the feature you want
> 
> W (^-^) (´｡･v･｡｀)(´｡･v･｡｀) im learning too. ill try too

I think you should learn kotlin, the language the app was built in
